engine/physics/src/physics/box2d/box2d_physics.cpp

```diff
@@ -1082,7 +1082,11 @@ namespace dmPhysics
             return;
         }
 
-        HullSet::Hull& hull = shape_data->m_HullSet->m_Hulls[shape_data->m_Cells[child].m_Index];
+        const GridShapeData::Cell& cell = shape_data->m_Cells[child];
+        if (cell.m_Index == B2GRIDSHAPE_EMPTY_CELL)
+            return;
```

Comment:
When the user sets tile "0", it means "clear the cell", and the index is set to B2GRIDSHAPE_EMPTY_CELL.
After this has happened, we didn't check it before trying to iterate over the hull vertices (cannot get a hull is it's an empty cell)